runtime/include/RuntimeCAPI.h

```diff
@@ -25,8 +25,9 @@ extern "C" {
 #endif
 
 // Quantum Runtime Instructions
-void __quantum__rt__fail_cstr(const char *cstr);
+void __quantum__rt__fail_cstr(const char *);
 void __quantum__rt__initialize();
+void __quantum__rt__device(int8_t *, int8_t *);
 void __quantum__rt__finalize();
 void __quantum__rt__toggle_recorder(bool);
 void __quantum__rt__print_state();
```

Comment:
The PR description mentions that the new runtime instruction can also be used (among other things) to toggle the tape recording, but I still see the `toggle_recorder` instruction here (which I don't mind since tape recording is not a static property of the device).

Do I interpret the instruction correctly that you can only set one property at a time with this instruction, rather than a full device specification at once (e.g. `specs = ["backend", "lightning.qubit", "shots", "1000", ...]`)?

Reply:
That was my original plan to use `__rt__device(spec, value)` method all of device specifications in the format of `spec=value`. But, the `toggle_recorder` feature is not really a specification of a device (it can dynamically change within the scope of the device). The description is outdated. 

You can add all of the specifications at once via the `quantum.device` op, but this would lowered to a list of `__rt__device` down the road. I simply used this signature for the clarity and simplicity of the device instruction. One can check, modify these device specifications independent of each other (that came useful for debugging purposes).

Reply:
Got it, thanks!